app/main.py

```diff
@@ -1,8 +1,88 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
The best approach when parameters don't fit in 1 line is to move them on the next lines after method name